gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java

```diff
@@ -54,36 +54,21 @@ public class GcpLogSink implements Sink {
       = ImmutableSet.of("project_id", "location", "cluster_name", "namespace_name",
       "pod_name", "container_name");
   private static final long FALLBACK_FLUSH_LIMIT = 100L;
+  private final String projectId;
   private final Map<String, String> customTags;
-  private final Logging gcpLoggingClient;
   private final MonitoredResource kubernetesResource;
   private final Long flushLimit;
+  private Logging gcpLoggingClient;
```

Comment:
nit: it will be good to add a comment about why we are doing lazy initialization of `gcpLoggingClient` .

Reply:
Added. PTAL